README.md

```diff
@@ -22,7 +22,7 @@ makes it easy to visualize pipelines running in production,
 monitor progress, and troubleshoot issues when needed.
 
 ## Getting started
-Please visit the Airflow Platform documentation for help with [installing Airflow](http://pythonhosted.org/airflow/installation.html), getting a [quick start](http://pythonhosted.org/airflow/start.html), or a more complete [tutorial](http://pythonhosted.org/airflow/tutorial.html).
+Please visit the Airflow Platform documentation for help with [installing Airflow](http://airflow.incubator.apache.org/installation.html), getting a [quick start](http://airflow.incubator.apache.org/start.html), or a more complete [tutorial](http://airflow.incubator.apache.org/tutorial.html).
```

Comment:
Please make it `https://` and I think it would be wise to explicitly mention that these docs are the latest and not the latest release.